test/screenshot/fixture.test.scss

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
At previous companies, I saw the term "fixture" used in screenshot tests to refer to the minimal UI that _hosts_ (wraps or contains) the components being tested.

The word _might_ be borrowed from [JUnit](https://github.com/junit-team/junit4/wiki/test-fixtures)? (Many of my coworkers were Java developers.)

> The purpose of a test fixture is to ensure that there is a well known and fixed environment in which tests are run so that results are repeatable.

I'm open to suggestions for a better name 😀